src/collar/oc_PluginTemplate.lsl

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
I think we should put this in spares/ rather than in the collar folder.  I'd like the collar folder to only have scripts in it that we actually need to import inworld.  That can make the copy/paste process faster when we do a release (or if we ever get a bot to automate it).

Reply:
Otherwise this looks worth merging.